src/firefly/js/ui/SuggestBoxInputField.css

```diff
@@ -0,0 +1,31 @@
+.SuggestBoxInputField {
+  position: relative;
+}
+
+.SuggestBoxPopup {
+    position: absolute;
+    overflow: auto;
+    top: 100%;
+    background: #fff;
+    z-index: 1;
+    max-height: 200px;
+    border: 2px solid #7eadd9;
+}
+
+.SuggestBox {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+.SuggestBox__Suggestion {
+  display: block;
+  padding: 2px 20px;
+  cursor: default;
+}
+
+.SuggestBox__Suggestion--highlighted {
+  outline: 0;
+  color: white;
+  background: #7eadd9;
+}
```

Comment:
Good use of BEM naming convention.  I've been converting over whenever I get a chance as well.